src/components/Tooltip/Layout.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {ReactNode} from 'react';
+import {block} from '../utils/cn';
+import type {DOMProps, QAProps} from '../types';
+import {HotKey, HotKeyProps} from '../HotKey';
+
+import './Layout.scss';
+
+const b = block('tooltip-layout');
+
+export interface TooltipLayoutProps extends DOMProps, QAProps {
+    title: string;
+    hotkey?: HotKeyProps['value'];
```

Comment:
I don't like that TooltipLayout knows about hotkey. Imo its better to introduce a new component based on original Tooltip - HotkeyTooltip